h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -183,23 +183,43 @@ protected void map(int id) {
         computeMetrics, 
         CFuncRef.from(_parms._custom_metric_func)
     );
+    ModelMetrics mmStackedEnsemble = null;
     if (computeMetrics) {
       // #score has just stored a ModelMetrics object for the (metalearner, preds_levelone) Model/Frame pair.
       // We need to be able to look it up by the (this, fr) pair.
       // The ModelMetrics object for the metalearner will be removed when the metalearner is removed.
       Key<ModelMetrics>[] mms = metalearner._output.getModelMetrics();
       ModelMetrics lastComputedMetric = mms[mms.length - 1].get();
-      ModelMetrics mmStackedEnsemble = lastComputedMetric.deepCloneWithDifferentModelAndFrame(this, fr);
+      mmStackedEnsemble = lastComputedMetric.deepCloneWithDifferentModelAndFrame(this, fr);
       this.addModelMetrics(mmStackedEnsemble);
       //now that we have the metric set on the SE model, removing the one we just computed on metalearner (otherwise it leaks in client mode)
       for (Key<ModelMetrics> mm : metalearner._output.clearModelMetrics(true)) {
         DKV.remove(mm);
       }
     }
     Frame.deleteTempFrameAndItsNonSharedVecs(levelOneFrame, adaptFrm);
-    return predictFr;
+    return new StackedEnsemblePredictScoreResult(predictFr, mmStackedEnsemble);
   }
 
+  private class StackedEnsemblePredictScoreResult extends PredictScoreResult {
+    private final ModelMetrics _modelMetrics;
+
+    public StackedEnsemblePredictScoreResult(Frame preds, ModelMetrics modelMetrics) {
+      super(null, preds, preds);
+      _modelMetrics = modelMetrics;
+    }
+
+    @Override
+    public ModelMetrics getOrMakeMetrics(Frame fr, Frame adaptFrm) {
+      return _modelMetrics;
+    }
+
+    @Override
+    public ModelMetrics.MetricBuilder<?> getOrMakeMetricBuilder(Frame adaptFrm) {
+      return StackedEnsembleModel.this.scoreMetrics(adaptFrm);
```

Comment:
I don't think you have to override this method. 

We don't have working implementation `StackedEnsembleModel#makeMetricBuilder` but also we don't have working implementation of `StackedEnsembleModel#score0` which is used by `BigScore#map` and using that by `Model#scoreMetrics`. 

Either way I think this call would not work so I'd rather keep it consistent with the parent class (and eventually make SE more like the rest of the models).

Reply:
@tomasfryda thanks for the comment, I get it now, there is no CV for Stacked Ensembles as such, there is CV on the meta learner level but not on the outer SE model

this means I can simplify the code quite a bit!